### ISSUE TITLE
Meet with NCSC web checks team

### ISSUE THREAD

**Issue by h701h** [canada-ca/tracker#3762]
As a project manager of Tracker 3.0 I want to understand the roadmap of the NCSC web checks team so we can borrow and contribute from their previous efforts. 

- [x] Get contacts
- [x] Set meeting date
- [x] Set agenda
  - [x]  TBD
  - [x] List of things they scan for?
  - [x] How their scanners are written?
  - [x] Roadmap?
  - [x] Sharing of code?

Things to review before we meet them:
- [x] TBD
- [x] https://www.ncsc.gov.uk/information/web-check

**Comment by h701h** [comment]
On hold until NCSC gets back to us. Request to meet sent. We are taking a top down approach to meeting. The bosses of both teams have to agree.

**Comment by h701h** [comment]
We met with the NCSC team. Waiting on accounts to login to their system.
